src/integration-test/system-integration-tests.ts

```diff
@@ -4,879 +4,305 @@
+          }
+        });
       });
-      if (ret.status === 400) {
-        log('Project was not yet fully deleted. Sleeping for two seconds.');
-        await sleep(2000);
-      } else {
-        expect(ret.status).to.equal(201);
-        const json = await ret.json();
-        expect(json.data.id).to.exist;
-        _repoUrl = json.data.attributes['repo-url'];
-        expect(_repoUrl).to.exist;
-        log(`Repository url for new project is ${_repoUrl}`);
-        _projectId = parseInt(json.data.id, 10);
-        expect(_projectId).to.exist;
-      }
-    }
-    log(`Project created (projectId: ${_projectId})`);
-    return { projectId: _projectId, repoUrl: _repoUrl };
-  }
 
-  async function shouldAllowForDeletingProject(_oldProjectId?: number) {
-    if (_oldProjectId) {
-      logTitle(`Deleting old integration-test-project (${_oldProjectId})`);
-      const ret = await adminFetch(`${charles}/api/projects/${_oldProjectId}`, {
-        method: 'DELETE',
+      describe('realtime', () => {
+        describe('team scoped events', () => {
+          const eventResponses: SSE[] = [];
+          const numEvents = 2;
+          const eventType = 'PROJECT_EDITED';
+          it('should be able to get realtime events', async function () {
+            for (let k = 0; k < numEvents; k++) {
+              // Arrange
+              const eventStream = await client.teamEvents(eventType);
+              const eventPromise = eventStream.take(1).toPromise();
+              const newDescription = 'fooo fooofoofoo bababa';
+
+              // Act
+              const editPromise = client.editProject({ description: newDescription });
+              const [editResponse, sseResponse] = await Promise.all([editPromise, eventPromise]);
+
+              // Assert
+              expect(sseResponse.type).to.equal(eventType);
+              expect(sseResponse.lastEventId).to.exist;
+              const event = JSON.parse(sseResponse.data);
+              expect(event).to.exist;
+              expect(event.id).to.eq(client.lastProject!.id);
+              expect(event.description).to.eq(newDescription);
+
+              expect(editResponse.id).to.exist;
+              expect(editResponse.attributes.description).to.equal(newDescription);
+              eventResponses[k] = sseResponse;
+            }
+          });
+
+          it('should be able to request events retrospectively', async function () {
+            const eventStream = await client.teamEvents(eventType, eventResponses[0].lastEventId);
+            const sseResponse = await eventStream.take(1).toPromise();
+
+            expect(sseResponse.type).to.equal(eventType);
+            expect(sseResponse.lastEventId).to.eq(eventResponses[1].lastEventId);
+            const event = JSON.parse(sseResponse.data);
+            expect(event).to.exist;
+            expect(event.id).to.eq(client.lastProject!.id);
+          });
+        });
+        describe.skip('deployment scoped events', () => {
```

Comment:
Yes. Tests for this endpoint didn't exist before and it needs some additional work to get the token.